### PR TITLE
Add flow test.

### PR DESCRIPTION
**Pull request by LShostenko**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -0,0 +1,83 @@
+require 'rake'
```
Comment (pcinkh) [inline Rakefile]: Remove rake

File: .coveragerc [hunk 2]
```diff
@@ -1,4 +1,5 @@
 [run]
 omit =
-    lexrank/mappings/*
     lexrank/settings.py
+    lexrank/tools/*
+    */__init__.py
```
Comment (pcinkh) [inline .coveragerc]: bring back init

File: lexrank/settings.py [hunk 3]
```diff
@@ -9,3 +9,9 @@ def get_folder(name):
 
 
 ASSETS_ROOT = get_folder('lexrank.assets')
+
+try:
```
Comment (pcinkh) [inline lexrank/settings.py]: move from here

File: lexrank/tools/assemble_stopwords.py [hunk 4]
```diff
@@ -23,11 +22,29 @@ def assemble_stopwords():
...
+    if not stopwords_map:
+        return
+
+    out_file = settings.ASSETS_ROOT / 'stopwords.json.gz'
+
     with gzip.open(out_file, mode='wt', encoding='utf-8') as fp:
         json.dump(stopwords_map, fp)
 
         msg = 'Stopwords written to {out_file}\n'.format(out_file=out_file)
         sys.stdout.write(msg)
```
Comment (pcinkh) [inline lexrank/tools/assemble_stopwords.py]: let us think about logger, create issue for future

Comment (LShostenko) [reply lexrank/tools/assemble_stopwords.py]: Done

**Pull request merged**